src/routes/docs/tutorials/nextjs-ssr-auth/step-3/+page.markdoc

```diff
@@ -17,7 +17,6 @@ import { cookies } from "next/headers";
 
 export async function createSessionClient() {
   const client = new Client()
-    .setEndpoint(process.env.NEXT_PUBLIC_APPWRITE_ENDPOINT)
```

Comment:
Other parts of the tutorial needs to be updated too since this is removed, right?